spec/timezone.html

```diff
@@ -354,11 +370,10 @@ <h1>TimeZoneFrom ( _identifier_ )</h1>
       <h1>GetOffsetNanosecondsFor ( _timeZone_, _absolute_ )</h1>
       <emu-alg>
         1. Let _getOffsetNanosecondsFor_ be ? Get(_timeZone_, *"getOffsetNanosecondsFor"*).
+        1. If _getOffsetNanosecondsFor_ is *undefined*, set _getOffsetNanosecondsFor_ to %Temporal.TimeZone.prototype.getOffsetNanosecondsFor%.
         1. Let _offsetNanoseconds_ be ? Call(_getOffsetNanosecondsFor_, _timeZone_, « _absolute_ »).
-        1. If Type(_offsetNanoseconds_) is not Number, then
-          1. Throw a *TypeError* exception.
-        1. If _offsetNanoseconds_ is not an integer Number value between −86400 × 10<sup>9</sup> and 86400 × 10<sup>9</sup> exclusive, then
-          1. Throw a *RangeError* exception.
+        1. If Type(_offsetNanoseconds_) is not Number, throw a *TypeError* exception.
```

Comment:
Btw, do we want this, or call `ToNumber`?

Reply:
Sounds fine, to me.